src/main/java/io/confluent/connect/hdfs/parquet/ParquetRecordWriterProvider.java

```diff
@@ -69,15 +70,15 @@ public void write(SinkRecord record) {
           try {
             log.info("Opening record writer for: {}", filename);
             org.apache.avro.Schema avroSchema = avroData.fromConnectSchema(schema);
-            writer = new AvroParquetWriter<>(
-                path,
-                avroSchema,
-                compressionCodecName,
-                blockSize,
-                pageSize,
-                true,
-                conf.getHadoopConfiguration()
-            );
+            writer = AvroParquetWriter.<GenericRecord>builder(path)
+                                      .withSchema(avroSchema)
```

Comment:
I believe the format in multiline chained calls is still 2 tabs in in connectors. Not a big deal though.